pkgs/applications/audio/drumgizmo/default.nix

```diff
@@ -7,7 +7,7 @@ stdenv.mkDerivation rec {
   name = "drumgizmo-${version}";
 
   src = fetchurl {
-    url = "http://www.drumgizmo.org/releases/${name}/${name}.tar.gz";
+    url = "https://www.drumgizmo.org/releases/${name}/${name}.tar.gz";
```

Comment:
Anyway, that's still strictly out of scope for this pr so I agree with leaving it out.